build.sbt

```diff
@@ -94,14 +92,6 @@ lazy val microservice = Project(appName, file("."))
         }.value)
     )
   )
-  .settings(ServiceManagerPlugin.serviceManagerSettings)
-  .settings(
-    itDependenciesList := List(
-      ExternalService("PREFERENCES"),
```

Comment:
They don’t  work with the latest service-manager - sm2. 
We can still run it:test by starting the necessary services manually.

Reply:
May be we should all use sm2 and this up for all services

Reply:
Good point @muralishub , for now I have put this plugin functionality back because I realised that the jenkins build also relies on it - so I was a bit too hasty to remove. However we need to find a solution for this issue.